src/app/components/Home/Home.jsx

```diff
@@ -15,9 +15,11 @@ const Home = ({ sortBy }) => (
 
     <div className="content-wrapper">
       <div className="nypl-column-three-quarters nypl-column-offset-one">
```

Comment:
inside the `nypl-full-width-wrapper` but _after_  the `nypl-row` would go the rest of the pages (in a row)